template.distillery/Makefile.style

```diff
@@ -45,12 +59,14 @@ endif
 # If SASS is not activated, it will concatenate all CSS files (listed in
 # $(CSS_FILES)) in $(LOCAL_CSS).
 # In both cases, external CSS files ($(EXTERNAL_CSS_FILES)) are copied.
-$(LOCAL_CSS): check_sass import-external-css
+$(LOCAL_CSS): check_sass check_postcss import-external-css
 ifeq "$(USE_SASS)" "yes"
 	[ -d $(SASSDIR) ] && \
 	(sassc -t compressed $(addprefix -I ,$(subst :, ,$(SASS_PATH))) $(SASS_SRC) $@ || \
 	SASS_PATH=$(SASS_PATH) sass --style compressed $(SASS_SRC) $@)
+	postcss --use autoprefixer --replace $@
 else
+        cp -f $(LOCAL_STATIC_DEFAULTCSS)/* $(LOCAL_STATIC_CSS)
```

Comment:
You replace tabs by more spaces. I think it would be better to replace it by tabs to keep the same styling code.